src/anaconda_cli_base/plugins.py

```diff
@@ -44,11 +50,18 @@ def load_registered_subcommands(app: Typer) -> None:
 
         if "login" in [cmd.name for cmd in subcommand_app.registered_commands]:
             auth_handlers[name] = subcommand_app
+            alias = AUTH_HANDLER_ALIASES.get(name, name)
+            auth_handlers[alias] = subcommand_app
```

Comment:
I'm having some bugs around .org. I think it's around here? 
![image](https://github.com/user-attachments/assets/912af509-d47c-4235-a12a-0b104b64b5cd)